baybe/constraints/continuous.py

```diff
@@ -117,16 +134,31 @@ def to_botorch(
         from baybe.utils.torch import DTypeFloatTorch
 
         param_names = [p.name for p in parameters]
-        param_indices = [
-            param_names.index(p) + idx_offset
-            for p in self.parameters
-            if p in param_names
-        ]
+        if not self.is_interpoint:
+            param_indices = [
+                param_names.index(p) + idx_offset
+                for p in self.parameters
+                if p in param_names
+            ]
+            coefficients = self.coefficients
+            torch_indices = torch.tensor(param_indices)
+        else:
+            assert (
+                batch_size is not None
+            ), "No batch_size set but using interpoint constraints"
```

Comment:
dont really like this error message, think of a suer getting this. they dont set anything as this is internal so this will confuse